src/NovelRT/ResourceManagement/Desktop/DesktopResourceLoader.cpp

```diff
@@ -327,4 +330,43 @@ namespace NovelRT::ResourceManagement::Desktop
         file.write(reinterpret_cast<const char*>(buffer.data()), std::streamsize(buffer.size()));
         file.close();
     }
+
+    AudioMetadata DesktopResourceLoader::LoadAudioFrameData(std::filesystem::path filePath)
+    {
+        if (filePath.is_relative())
+        {
+            filePath = _resourcesRootDirectory / "Audio" / filePath;
+        }
+
+        const size_t _bufferSize = 2048;
+        SF_INFO info;
+        info.format = 0;
+        auto filePathString = filePath.string();
+        SNDFILE* file = sf_open(filePathString.c_str(), SFM_READ, &info);
+
+        if (file == nullptr)
+        {
+            throw NovelRT::Exceptions::FileNotFoundException(filePath.string());
```

Comment:
Is this entirely true here? - the original logging would log the error message returned by sndfile (whether it was malformed file, wrong/unsupported format, etc) so we should probably do the same before throwing (or we include sndfile's error in the exception we throw)